### DISCUSSION TITLE
Running out of buffer for large models

### DISCUSSION THREAD

**Discussion by MariusGalyan** [abetlen/llama-cpp-python#361] [Q&A]
I am getting this error message running an 18GB large 30B parameter llm model:
buffer 'data' size 18300780544 is larger than buffer maximum of 17179869184
llama_init_from_file: failed to add buffer

Other llm models with smaller sizes run without any problems. I have a Mac Metal...

**Comment by jphme** [comment]
I have the same problem and openend an Issue here: https://github.com/ggerganov/llama.cpp/issues/1815

**Reply by MariusGalyan** [reply]
thank you!
